makefiles/Makefile.docs-bi-connector

```diff
@@ -29,7 +29,12 @@ html: ## Builds this branch's HTML under build/<branch>/html
 next-gen-html:
 	# snooty parse and then build-front-end
 	curl https://raw.githubusercontent.com/mongodb/docs-worker-pool/meta/publishedbranches/docs-bi-connector.yaml > ${REPO_DIR}/published-branches.yaml
-	@-echo ${SNOOTY_DB_PWD} | snooty build "${REPO_DIR}" "mongodb+srv://${SNOOTY_DB_USR}:@cluster0-ylwlz.mongodb.net/snooty?retryWrites=true" --commit "${COMMIT_HASH}" || exit 0;
+	@echo ${SNOOTY_DB_PWD} | snooty build "${REPO_DIR}" "mongodb+srv://${SNOOTY_DB_USR}:@cluster0-ylwlz.mongodb.net/snooty?retryWrites=true" --commit "${COMMIT_HASH}";\
```

Comment:
We've typically condensed makefile scripting instead of having it span multiple lines and that prevents the targets from bloating beyond what a terminal screen will show at once. Will this work condensed into one line? If so let's stick to that convention for now.

Reply:
I could unfortunately not get it to work in one line! I believe conditionals in make are something that only work with `\` and new lines. It seems to be very sticky!